[2024-09-11T02:38:04.464+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T02:38:04.529+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-03T00:00:00+00:00 [queued]>
[2024-09-11T02:38:04.556+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-03T00:00:00+00:00 [queued]>
[2024-09-11T02:38:04.560+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T02:38:04.598+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-07-03 00:00:00+00:00
[2024-09-11T02:38:04.615+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=18083) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T02:38:04.616+0000] {standard_task_runner.py:63} INFO - Started process 18093 to run task
[2024-09-11T02:38:04.623+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2024-07-03T00:00:00+00:00', '--job-id', '3953', '--raw', '--subdir', 'DAGS_FOLDER/13_dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmplqh8nyle']
[2024-09-11T02:38:04.635+0000] {standard_task_runner.py:91} INFO - Job 3953: Subtask insert_into_table
[2024-09-11T02:38:04.729+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-07-03T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T02:38:04.912+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-03T00:00:00+00:00'
[2024-09-11T02:38:04.915+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T02:38:04.951+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-07-03', 'dag_with_postgres_operator_v03')
        
[2024-09-11T02:38:04.971+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T02:38:04.990+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T02:38:05.000+0000] {sql.py:487} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-07-03', 'dag_with_postgres_operator_v03')
        , parameters: None
[2024-09-11T02:38:05.004+0000] {sql.py:496} INFO - Rows affected: 1
[2024-09-11T02:38:05.010+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T02:38:05.021+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, run_id=scheduled__2024-07-03T00:00:00+00:00, execution_date=20240703T000000, start_date=20240911T023804, end_date=20240911T023805
[2024-09-11T02:38:05.121+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T02:38:05.151+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-11T02:38:05.159+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
